src/Adapter/SqlManager/SqlQueryValidator.php

```diff
@@ -84,10 +84,6 @@ private function getErrors(array $sqlErrors)
         $errors = [];
 
         foreach ($sqlErrors as $key => $sqlError) {
```

Comment:
Could you add unit tests for this behavior change ?

Reply:
I can't, it's an ObjectModel validator :(